sklearn/model_selection/_search.py

```diff
@@ -4,40 +4,44 @@
 """
 from __future__ import print_function
 
-# Author: Alexandre Gramfort <alexandre.gramfort@inria.fr>,
```

Comment:
Actually, it was the organise imports function in Eclipse that did this. I'll put these back as was with the next push and then disable that option.